### ISSUE TITLE
Warning about Sessionmakers not containing handle even though handle is given

### ISSUE THREAD

**Issue by blueandgold** [GoogleCloudPlatform/forseti-security#1199]
ubuntu@forseti-client-2207-vm:~$ forseti model use 55724594f377ec5f41f4d57e558a44a8; forseti scanner run

```
A  Mar  2 22:15:37 forseti-server-2207-vm systemd[1]: Stopped Forseti API Server. 
A  Mar  2 22:15:37 forseti-server-2207-vm systemd[1]: Started Forseti API Server. 
A  Mar  2 22:15:38 forseti-server-2207-vm cloud_sql_proxy[22036]: 2018/03/02 22:15:38 New connection for "forseti-196022:us-central1:forseti-server-2207" 
A  [forseti-security] INFO google.cloud.forseti.services.explain.service(create_and_register_service): Service <google.cloud.forseti.services.explain.service.GrpcExplainer object at 0x7f73cd765ed0> created and registered. 
A  [forseti-security] INFO google.cloud.forseti.services.model.service(create_and_register_service): Service <google.cloud.forseti.services.model.service.GrpcModeller object at 0x7f73cc6f44d0> created and registered 
A  [forseti-security] INFO google.cloud.forseti.services.scanner.service(create_and_register_service): Service <google.cloud.forseti.services.scanner.service.GrpcScanner object at 0x7f73cc6f46d0> created and registered. 
A  [forseti-security] INFO google.cloud.forseti.services.notifier.service(create_and_register_service): Service <google.cloud.forseti.services.notifier.service.GrpcNotifier object at 0x7f73cc6f47d0> created and registered 
A  Mar  2 22:15:57 forseti-server-2207-vm cloud_sql_proxy[22036]: 2018/03/02 22:15:57 New connection for "forseti-196022:us-central1:forseti-server-2207" 
A  [forseti-security] INFO google.cloud.forseti.services.scanner.service(Run): Run scanner service with model: 55724594f377ec5f41f4d57e558a44a8 
A  [forseti-security] INFO google.cloud.forseti.scanner.scanner_builder(build): Initializing the rules engine: 
A  [forseti-security] INFO google.cloud.forseti.scanner.scanner_builder(build): Initializing the rules engine: 
A  [forseti-security] WARNING google.cloud.forseti.services.dao(_get): Sessionmakers doesn't contain handle, handle = 55724594f377ec5f41f4d57e558a44a8 
```

**Comment by blueandgold** [comment]
This is really not a warning, but rather an info or debug level to say that the handle is not found and will be created.

**Comment by joecheuk** [comment]
Resolved #1250
